WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -832,16 +836,24 @@ private void SetFeeTargetLimits()
 			}
 		}
 
-		private bool TryParseUserFee(out decimal userFee)
+		private bool TryParseUserFee(out decimal userFeeRateAsDecimal)
 		{
-			userFee = default;
-			var userFeeText = UserFeeText;
-			return
-				userFeeText is { }
-				&& !userFeeText.Contains(",")
-				&& decimal.TryParse(userFeeText, out userFee)
-				&& (userFee * 1_000) < Constants.MaxSatoshisSupply
-				&& userFee > 0;
+			try
+			{
+				if (decimal.TryParse(UserFeeText, NumberStyles.Integer, new CultureInfo("en-US"), out var userFee))
+				{
+					var userFeeRate = new FeeRate(userFee);
+					userFeeRateAsDecimal = userFeeRate.SatoshiPerByte;
+					return userFeeRate > MinRelayTxFeeRate && userFeeRate <= AbsurdlyHighFeeRate;
```

Comment:
Shouldn't this be `return userFeeRate >= MinRelayTxFeeRate && userFeeRate <= AbsurdlyHighFeeRate;`?

Reply:
No, MinRelayTxFeeRate is zero, you can't have 0 sats/vbyte right?